base/gmp.jl

```diff
@@ -664,7 +660,7 @@ powermod(x::Integer, p::Integer, m::BigInt) = powermod(big(x), big(p), m)
 
 function gcdx(a::BigInt, b::BigInt)
     if iszero(b) # shortcut this to ensure consistent results with gcdx(a,b)
-        return a < 0 ? (-a,-ONE,b) : (a,one(BigInt),b)
+        return a < 0 ? (-a,-ONE(),b) : (a,one(BigInt),b)
```

Comment:
Huh, what does that do -- `ONE` still is a `BigInt` here, isn't it? And those are not callable?

Reply:
Darn, this slipped through.  Good catch.